### PR TITLE
Add Uber mobile app as performance test project

### PR DESCRIPTION
**Pull request by wolfs**
So we can run ad-hoc performance tests against the project. We should run a test against the app in Ready for Release, aka once a day.

### CODE REVIEW

**Review by lptr — COMMENTED** [review]
LGTM, left a couple comments.

File: build-logic/performance-testing/src/main/groovy/gradlebuild.performance-templates.gradle [hunk 1]
```diff
@@ -401,6 +401,19 @@ performanceTest.registerTestProject("santaTrackerAndroidBuild", RemoteProject) {
...
+    remoteUri = 'https://github.com/gradle/android-build-eval.git'
+    subdirectory = 'mobile_app1'
+    // Pinned from PR https://github.com/uber-common/android-build-eval/pull/17
+    ref = '18ff193198010064e43e9244edde91ab54c2da70'
+    def fileSystemOperations = gradle.services.get(FileSystemOperations)
+    doLast {
+        fileSystemOperations.delete {
+            delete(new File(outputDirectory, ".okbuck"))
+        }
+    }
```
[the remark this answers is not in the document]
Comment (wolfs) [reply build-logic/performance-testing/src/main/groovy/gradlebuild.performance-templates.gradle:414]: What do you suggest I use? Project.delete is not CC compatible.

[the remark this answers is not in the document]
Comment (lptr) [reply build-logic/performance-testing/src/main/groovy/gradlebuild.performance-templates.gradle:414]: Interesting. Let's leave this as it is then. I asked about why the CC compatible way is this complex.

File: subprojects/internal-integ-testing/src/main/groovy/org/gradle/internal/scan/config/fixtures/ApplyGradleEnterprisePluginFixture.groovy [hunk 2]
```diff
@@ -29,7 +29,13 @@ class ApplyGradleEnterprisePluginFixture {
     """
 
     static void applyEnterprisePlugin(File settingsFile) {
-        prefixFile(settingsFile, APPLY_ENTERPRISE_PLUGIN)
+        def settingsText = settingsFile.text
+        def matcher = settingsText =~ /id[ (]"com.gradle.enterprise"[)]? version[ (]["'](.*)["'][)]?/
```
[the remark this answers is not in the document]
Comment (wolfs) [reply subprojects/internal-integ-testing/src/main/groovy/org/gradle/internal/scan/config/fixtures/ApplyGradleEnterprisePluginFixture.groovy]: Yes, fixed.

**Comment by wolfs** [comment]
@bot-gradle test QFL

**Review by lptr — APPROVED** [review]
LGTM.

**Comment by wolfs** [comment]
@bot-gradle test QFL

**Comment by wolfs** [comment]
@bot-gradle test QFL

**Pull request merged**